src/GlobalStyle.tsx

```diff
@@ -0,0 +1,20 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+  * {
+    box-sizing: border-box;
+  }
+  html {
+    font-size: 62.5%;
+  }
+  body {
+    font-family: 'Spoqa Han Sans Neo';
+    font-size: 1.6rem;
+  }
```

Comment:
See also:
- [Why I Change the Font Size to 62.5% in Every Project](https://javascript.plainenglish.io/why-i-change-the-font-size-to-62-5-in-every-project-45c5ff785fb5)